tests/overlay_tests.cpp

```diff
@@ -1347,6 +1349,13 @@ TEST_F(OverlayTest, ROOT_checkAddVirtualNetworks)
   ASSERT_EQ(agentOverlay->info().subnet(), "11.0.0.0/8");
   ASSERT_EQ(agentOverlay->subnet6(), "fd04::/80");
   ASSERT_EQ(agentOverlay->info().subnet6(), "fd04::/64");
+
+  // cleanup the network
+  Future<string> docker = runCommand("docker",
```

Comment:
This cleanup should ideally in both TearDown and SetUp in case someone ctrl-c the test.

Reply:
The reason I kept it here is that this network is created by this test.

Reply:
Yeah, i know. I think we can make this network name another constant, and always try to cleanup in the TearDown or SetUp (and ignore errors)